src/anki/ankiclient.cpp

```diff
@@ -1765,6 +1780,83 @@ QString AnkiClient::buildFrequencies(const QList<Frequency> &frequencies)
     return freqStr;
 }
 
+std::vector<int> AnkiClient::getFrequencyNumbers(
+        const QList<Frequency> &frequencies)
+{
+    QString previousDictionary;
+    std::vector<int> frequencyNumbers;
+
+    for (const auto& frequencyEntry : frequencies)
```

Comment:
The type is short enough here I don't think you should use `auto`. Also I prefer my `&` next to the variable.

Reply:
Fixed in https://github.com/ripose-jp/Memento/pull/238/commits/a92c0c0854dbf417536134dedbb79e2bc0812077